src/test/ui/parser/raw-byte-string-literals.rs

```diff
@@ -2,6 +2,7 @@
 
 
 pub fn main() {
+    br"a
"; //~ ERROR bare CR not allowed in string
     br"é";  //~ ERROR raw byte string must be ASCII
     br##~"a"~##;  //~ ERROR only `#` is allowed in raw string delimitation
 }
```

Comment:
Let's also add a test that \r\n is translated to \n?

I guess a run-pass `assert_eq!(rb"\r\n", [10])` should do the trick

Reply:
As it turns out we already have a test for that:
https://github.com/rust-lang/rust/blob/5c6f6b810c1d5850814a3c5960c4dfe2aee77d0d/src/test/run-pass/lexer-crlf-line-endings-string-literal-doc-comment.rs#L30-L32

Reply:
That tests raw strings, and not raw *byte* strings. Let's extend that test to raw byte strings as well: after all, this is the bug we want to fix with this change :D

Reply:
Right :man_facepalming: Adding a test now